RecoCTPPS/TotemRPLocal/python/totemTimingLocalReconstruction_cff.py

```diff
@@ -7,7 +7,15 @@
 totemTimingRecHits.smoothingPoints = cms.int32(20)
 totemTimingRecHits.lowPassFrequency = cms.double(0.7)
 totemTimingRecHits.hysteresis = cms.double(5e-3)
+totemTimingRecHits.calibrationFile = cms.FileInPath('RecoCTPPS/TotemRPLocal/data/timing_offsets_ufsd_2018.dec18.cal.json')
+totemTimingRecHits.mergeTimePeaks = cms.bool(True);
```

Comment:
please remove all overrides which are already defined in totemTimingRecHits_cfi.
Unless there is a good reason to have these overrides, it would be best to simply move all defaults to their proper place in the fillDescriptions.